### DISCUSSION TITLE
Floating Node Links

### DISCUSSION THREAD

**Discussion by GenerallyDisjointed** [freeplane/freeplane#497] [Help]
Hi all

How do I link a floating node to a normal node? The auto adjustment/moving of nodes is all over the place for me, so am using floating instead

**Comment by matar3** [comment]
Create it as a normal child node of the normal node you want to link it to. Then, change the style of the child node to "floating node" (select the node and go to Format > Style and choose "floating node"). In this way you'll have a floating node linked to the parent node.
